packages/plugins/plugin-sheet/src/components/Sheet/util.ts

```diff
@@ -54,3 +63,34 @@ export const scrollIntoView = (scrollContainer: HTMLElement, el: HTMLElement) =>
     scrollContainer.scrollLeft += 2 + scrollerBounds.right - cellBounds.right;
   }
 };
+
+export type UseSheetModelProps = {
+  sheet: SheetType;
+  space: Space;
+  options?: Partial<FunctionContextOptions>;
+  readonly?: boolean;
+};
+
+export const useSheetModel = ({ space, sheet, options, readonly }: UseSheetModelProps) => {
+  const graph = useComputeGraph(space, options);
+
+  const [[model, formatting] = [], setModels] = useState<[SheetModel, FormattingModel] | undefined>(undefined);
+
```

Comment:
i like to remove the space here to indicate that the effect is the setter of the state. (phrasing).

Reply:
Which space do you mean? I believe I copied this code verbatim…